package.json

```diff
@@ -4,8 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "webpack-dev-server --open --mode development",
-    "build": "webpack --mode production"
+    "start": "webpack-dev-server --open --env.mode=development",
+    "build": "webpack --env.mod=production"
```

Comment:
what should be here env.mod or env.mode?

Reply:
added 'e'